Release/include/cpprest/containerstream.h

```diff
@@ -113,7 +113,7 @@ class basic_container_buffer : public streams::details::streambuf_state_manager<
 
         msl::safeint3::SafeInt<size_t> readhead(m_current_position);
         msl::safeint3::SafeInt<size_t> writeend(m_data.size());
-        return (size_t)(writeend - readhead);
+        return (size_t)writeend - (size_t)readhead;
```

Comment:
this and other changes like generated complaints about the implicit copy constructor.  This should be more efficient anyway.